GitUI/UserControls/RevisionGrid/Columns/MessageColumnProvider.cs

```diff
@@ -205,10 +204,9 @@ public override bool TryGetToolTip(DataGridViewCellMouseEventArgs e, GitRevision
 
                 var arrowType = gitRef.IsSelected
                     ? RefArrowType.Filled
-                    : gitRef.IsSelectedHeadMergeSource
```

Comment:
I would like to keep these, but they may not be working as they should now.

Reply:
https://github.com/gitextensions/gitextensions/pull/8900#discussion_r587810524